Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -148,6 +154,26 @@ class FirefoxHomeViewController: UICollectionViewController, HomePanel {
     }()
 
     var pocketStories: [PocketStory] = []
+    
+    var hasRecentBookmarks = false
+    var isRecentlySavedSectionEnabled: Bool {
+        get {
+            profile.places.getRecentBookmarks(limit: 5).uponQueue(.main) { [weak self] result in
```

Comment:
Since we're accessing `isRecentlySavedSectionEnabled` inside the UICollectionView lifecycle we should probably cache this value somewhere. That code gets called a lot 😬

Reply:
We will create a ticket to refactor this, along with other parts of FirefoxHomeVC to move towards MVVM.